File: Python/8py 3/plural.py
"""
|--------------------------------------------------------------------------
| Plural
|--------------------------------------------------------------------------
|
"""

def plural(n):
   return n != 1


def plural(n):
   return False if n == 1 else True


def plural(n):
   return not(n == 1)


def plural(n):
   return [True, False][n == 1]


print(plural(0), True)
print(plural(1), False)
print(plural(100), True)

File: JavaScript/8js 2/plural.js
/*
|--------------------------------------------------------------------------
| Plural
|--------------------------------------------------------------------------
|
*/

function plural(n) {
   return n != 1;
}


function plural(n) {
   return n == 1 ? false : true;
}


function plural(n) {
   return !(n == 1);
}


console.log(plural(0), true);
console.log(plural(0.5), true);
console.log(plural(1), false);
console.log(plural(100), true);
console.log(plural(Infinity), true);